src/components/CakeItems.tsx

```diff
@@ -102,14 +102,11 @@ export default function CakeItems({
         } justify-center items-baseline hover:cursor-pointer`}
         {...other}
       >
-        {Object.values(displayItems)?.map((item: Refreshment) => (
-          <ProductCard
+        {Object.values(displayItems)?.map((item: Refreshment | Cake) => (
+          <RefreshmentCard
             key={item.id}
-            name={item.name}
+            item={type === "CAKE" ? (item as Refreshment) : (item as Cake)}
```

Comment:
does it need to have "as" this line. if u can remove, pls do so. If not, just leave it like that

Reply:
actually imma create another card for cakes. over here we're checking because we share them between cake and refreshment